spec/javascripts/packages/document-capture/components/form-steps-spec.jsx

```diff
@@ -399,16 +391,6 @@ describe('document-capture/components/form-steps', () => {
     expect(inputOne.hasAttribute('data-is-error')).to.be.true();
   });
 
-  it('renders and moves focus to step errors', () => {
-    const steps = [STEPS[1]];
-
-    const { getByRole } = render(<FormSteps steps={steps} />);
-    const button = getByRole('button', { name: 'Create Step Error' });
```

Comment:
The corresponding button in the `STEPS` example content existed only for this test case, so could be removed as well.

Reply:
Good catch. I'll remove it.